Minor changes.

luciof55 opened the pull request:
Two configuration option were added. 

The first allows choose which field use to set issues's estimated hour (Work or Duration).
Second allow decide if summary task are fully imported including or not the following fields: estimated hour, start date, end date and done ratio.

Also field done ratio now is imported using "**PercentWorkComplete**" instead of "**PercentComplete**"

Commit:
Two configuration option were added. The first allows choose which field use to set issues's estimated hour (Work or Duration).
Second allow decide if summary task are fully imported including or not the following fields: estimated hour, start date, end date and done ratio.
Also field done ratio now is imported using "PercentWorkComplete" instead of "PercentComplete"

luciof55 commented:
Hi @braini75 

Please review this pull request.

Thanks in advance.

braini75 commented:
Hi luciof55,
thank you!
I will review your modifications! I need some time as I will be on vacation the next view weeks ;)
Greetings,
braini75

Commit:
Add option to delete all issue before import
Finish manually asigment mapping
Add Project parent issue feature
Interface improvements
Find user by login
Integrate with Plus Gantt Plugins
Mapping OutlineLevel attribute
Minor corrections

Commit:
Uselower case to compare user login and resource name.
Add Cancel button
Create parent issue even project does not have Name element.

Commit:
Add progress bar

braini75 committed:
Merge branch 'master' into master
